pkg/frontend/admin_openshiftcluster_mustgather.go

```diff
@@ -40,8 +40,15 @@ func (f *frontend) _postAdminOpenShiftClusterMustGather(ctx context.Context, w h
 		return err
 	}
 
-	w.Header().Add("Content-Type", "application/gzip")
-	w.Header().Add("Content-Disposition", `attachment; filename="must-gather.tgz"`)
```

Comment:
Moved to right before `io.Copy` to be consistent with https://github.com/Azure/ARO-RP/pull/886#discussion_r463724754